modules/admin_manual/pages/release_notes.adoc

```diff
@@ -19,6 +20,162 @@
+Additionally, PHP 7.3 support will be available in an upcoming version.
+
+If you're still running PHP 5.6, **you must upgrade to PHP 7 before upgrading to ownCloud Server 10.2**. 
+Please be aware that apps that do not support outdated PHP versions will not upgrade.
+
+TIP: See the xref:installation/system_requirements.adoc#officially-recommended-supported-options[system requirements in the ownCloud documentation]. 
+
+To allow for additional upgrade time, version 10.2 still supports PHP 7.0, because some of the major Linux distributions continue to support it. 
+However, support for PHP 7.0 will be discontinued in an upcoming version of ownCloud 10, to enhance both security and performance. 
+To prepare for this change, we strongly encourage you to begin planning an upgrade as soon as possible.
+
+=== Advanced Sharing Permissions
+
+The new server version https://github.com/owncloud/core/issues/34951[introduces the means] for extensions to implement additional, advanced permissions for user and group sharing. This feature increases sharing flexibility and opens the doors for extension developers to introduce new functionality based on sharing permissions.
+
+Especially, considering collaborative editing solutions, this addition provides the foundation for mode-based document sharing, such as "view-only", "comments-only" or "enforce change tracking". In the future, such advanced permissions should significantly improve the security as well as the usability of review processes, working on Office documents collaboratively, or exchanging information securely.
+
+Based on the new capabilities a set of features has been developed together with Collabora Online, called _Secure View_. Secure View is designed to enable information distribution processes for sensitive data, meaning that information can be provided securely yet can — *under no circumstances* — leave the platform.
+
+Practically, it enables users to share documents (such as docx, xlsx, pptx, and PDF files) in such a way that the recipient can't edit, download, copy and paste, nor print them. 
+Additional protection for screenshots and photos is provided by watermarks which display user information. 
+What's more, users can decide to allow printing and exporting of documents protected by watermarks as well.
+
+=== More Granular Permissions for Public Links on Folders
+
+With Server 10.2 a new permission for public links on folders is available for users. 
+It lets recipients view, download, and upload documents, yet prevents them from making changes (e.g., rename, move, collaborative editing). 
+The new permission, named `Download / View / Edit`, has been added as a replacement for the former `Download / View / Upload`.
+It has been changed and can now be regarded as a public file drop to distribute and gather information with a single link, while preventing recipients from changing contents.
```

Comment:
I reworked that paragraph. Are you okay with it now?

Reply:
No. You now have 4 permissions for public links on folders:
![Screenshot from 2019-05-08 16-47-46](https://user-images.githubusercontent.com/21193500/57384497-0b0c0d00-71b1-11e9-9ee8-a4656745f294.png)

- "Download / View / Edit" equals what earlier versions had as "Download / View / Upload"
- "Download / View / Upload" is new in a sense that you can just download and upload to such folders, not make any other changes like rename, delete, update, etc.

To make it more clear: Previously you only had "Download / View / Upload" which (in contradiction to its name) gave you also the permission to edit the folder's contents.

Reply:
btw. with the `"Download / View / Upload"` permission its possible to overwrite files, and by that `Edit` them. It think that was discussed somewhere else CC @sharidas 
Feel like we should at least document it

Reply:
`Download/View/Upload` permission will not allow to delete the files. Also if the user tries to uplaod same file name ( which is present in the oC ), then conflict dialog appears in the UI and user can opt to select whether to keep the original or the new one uploaded or both. rename files and download are available for this option.

Reply:
@pmaier1, the text has been further reworked. Can you confirm if it's suitable, or if it still isn't correct.

Reply:
> with the "Download / View / Upload" permission its possible to overwrite files, and by that Edit them

> rename files and download are available for this option.

Yeah, well. Then the implementation is wrong. Where's the sense in this? :man_facepalming: 

This is what's stated in the [ticket](https://github.com/owncloud/enterprise/issues/2742):

> As a user I want to share files/folders via public link. The recipient should be able to view, download and upload into such shared spaces. Still I want to prevent the recipient being able to edit (e.g. Collabora, move, rename, etc.) or delete contents.

Reply:
according to the technical spec here https://github.com/owncloud/enterprise/issues/2742#issuecomment-467349676 it was said that:
> Download / View / Upload ( no edit and delete possible )
> Download / View / Edit ( no upload possible but edit and delete possible )

so if edit is possible in "download / view / upload" (overwrite being an edit) then this needs fixing.

@individual-it can you raise a ticket for 10.2 ?

Reply:
Wait:

> Download / View / Edit ( no upload possible but edit and delete possible )

"Edit" includes "create", so upload must be possible with this option.

Reply:
- "Download / View / Upload": I can only read content and add additional content that doesn't already exist (create)
- "Download / View / Edit": I can read content, update it, rename it, move it, delete it, create/upload it